tests/bson/bson-packedarray-fromJSON-001.phpt

```diff
@@ -10,6 +10,8 @@ $tests = [
     '[ 1, 2, 3 ]',
     '[[ 1, 2, 3 ]]',
     '[{ "bar": 1 }]',
+    // JSON objects are accepted iff the keys are valid
```

Comment:
I was gonna nitpick the typo, but TIL that [iff is an actual word](https://en.wiktionary.org/wiki/iff).

Reply:
Yes, I don't like how easily this can be mixed up for a typo but it'd be a complete waste of space to write "if and only if"